libs/ledger-live-common/src/families/filecoin/bridge/account.ts

```diff
@@ -142,8 +142,8 @@ const estimateMaxSpendable = async ({
 
   if (balance.lte(estimatedFees)) return new BigNumber(0);
 
-  balance.minus(estimatedFees);
-  if (amount) balance.minus(amount);
+  balance = balance.minus(estimatedFees);
+  if (amount) balance = balance.minus(amount);
```

Comment:
Good catch, could you consider this as an opportunity to unit test this function?
We use jest, you'll find plenty of examples in the codebase, including ways to mock functions and backend calls.

Reply:
I will try to add a new test case for this hammar